internal/tiltfile/encoding/json.go

```diff
@@ -40,12 +41,22 @@ func readJSON(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple
 
 // reads json from a string
 func decodeJSON(thread *starlark.Thread, fn *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
-	var contents starlark.String
+	var contents starlark.Value
 	if err := starkit.UnpackArgs(thread, fn.Name(), args, kwargs, "json", &contents); err != nil {
 		return nil, err
 	}
 
-	return jsonStringToStarlark(contents.GoString(), "")
+	var s string
```

Comment:
this should be equivalent to value.AsString(s)?

Reply:
oh cool, I looked for something to handle the blob/string union and was surprised I couldn't find it